plugins/inputs/fibaro/fibaro.go

```diff
@@ -0,0 +1,183 @@
+package fibaro
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+	"github.com/influxdata/telegraf/plugins/inputs/httpjson"
+)
+
+const sampleConfig = `
+  ## Required Fibaro controller address/hostname.
+  ## Note: connection is done over http/80 as Fibaro did not implement https.
+  server = "<controller>"
+  ## Required credentials to access the API (http://<controller/api/<component>)
+  username = "<username>"
+  password = "<password>"
+`
+
+const description = "Read devices value(s) from a Fibaro controller"
+
+// Fibaro contains connection information
+type Fibaro struct {
+	Server   string
+	Username string
+	Password string
```

Comment:
Add a configurable timeout for the http.Client using internal.Duration:

https://github.com/influxdata/telegraf/blob/master/plugins/inputs/http/http.go#L37